src/docs/resources/platform-adaptations.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Platform Specific Behaviors and Adaptations
+---
+
+## Adaptation Philosophy
+
+There are generally 3 categories of platform adaptiveness:
+
+1. Things that are behaviors of the OS environment (such as text editing and
+scrolling) and that would be 'wrong' a different behavior took place.
+2. Things that are conventionally implemented in apps using the OEM's SDKs
+(such as using parallel tabs on iOS or showing an [android.app.AlertDialog](https://developer.android.com/reference/android/app/AlertDialog.html)
+on Android).
+3. Correct usage of corresponding platform APIs to export to TalkBack/VoiceOver,
+status bar color theme etc. and to import notch metrics, locale settings etc.
+
+This article mainly covers the automatic adaptations provided by Flutter
+in case 1 on Android and iOS.
+
+For case 2, Flutter bundles the means to produce the appropriate effects of
+the platform conventions but does not adapt automatically when app design
+choices are needed. For a discussion, see [#8410](https://github.com/flutter/flutter/issues/8410#issuecomment-468034023).
+
+Flutter should transparently work for scenario 3. 
+
+## Page Navigation
+
+### Navigation Transitions
+
+On **Android**, the default [Navigator.push](https://docs.flutter.io/flutter/widgets/Navigator/push.html)
+transition is modeled after [startActivity()](https://developer.android.com/reference/android/app/Activity.html#startActivity(android.content.Intent))
+which generally has one bottom-up animation variant.
+
+On **iOS**:
+
+* The default [Navigator.push](https://docs.flutter.io/flutter/widgets/Navigator/push.html)
```

Comment:
As in the other PR, globally change docs.flutter.io to api.flutter.dev.

Reply:
Actually let me correct myself. We don't use explicit URLs for API docs. Please use {{site.api}}/flutter/widgets/...

(This macro is relatively new, but it used all over the website.)

Reply:
done